types/culori/index.d.ts

```diff
@@ -1,280 +1 @@
-    modeLab,
-    modeLab65,
-    modeLch,
-    modeLch65,
-    modeLchuv,
-    modeLrgb,
-    modeLuv,
-    modeOkhsl,
-    modeOkhsv,
-    modeOklab,
-    modeOklch,
-    modeP3,
-    modeProphoto,
-    modeRec2020,
-    modeRgb,
-    modeXyb,
-    modeXyz50,
-    modeXyz65,
-    modeYiq,
-};
-
-export const a98: ConvertFn<"a98">;
-export const cubehelix: ConvertFn<"cubehelix">;
-export const dlab: ConvertFn<"dlab">;
-export const dlch: ConvertFn<"dlch">;
-export const hsi: ConvertFn<"hsi">;
-export const hsl: ConvertFn<"hsl">;
-export const hsv: ConvertFn<"hsv">;
-export const hwb: ConvertFn<"hwb">;
-export const jab: ConvertFn<"jab">;
-export const jch: ConvertFn<"jch">;
-export const lab: ConvertFn<"lab">;
-export const lab65: ConvertFn<"lab65">;
-export const lch: ConvertFn<"lch">;
-export const lch65: ConvertFn<"lch65">;
-export const lchuv: ConvertFn<"lchuv">;
-export const lrgb: ConvertFn<"lrgb">;
-export const luv: ConvertFn<"luv">;
-export const okhsl: ConvertFn<"okhsl">;
-export const okhsv: ConvertFn<"okhsv">;
-export const oklab: ConvertFn<"oklab">;
-export const oklch: ConvertFn<"oklch">;
-export const p3: ConvertFn<"p3">;
-export const prophoto: ConvertFn<"prophoto">;
-export const rec2020: ConvertFn<"rec2020">;
-// tslint:disable-next-line:use-default-type-parameter
-export const rgb: ConvertFn<"rgb">;
-export const xyb: ConvertFn<"xyb">;
-export const xyz50: ConvertFn<"xyz50">;
-export const xyz65: ConvertFn<"xyz65">;
-export const yiq: ConvertFn<"yiq">;
-
-/* Types */
-
-export type { A98 } from "./src/a98/types";
-export { Color, Gamut, GamutMode, Mode } from "./src/common";
-export type { Cubehelix } from "./src/cubehelix/types";
-export type { Dlab } from "./src/dlab/types";
-export type { Dlch } from "./src/dlch/types";
-export type { Hsi } from "./src/hsi/types";
-export type { Hsl } from "./src/hsl/types";
-export type { Hsv } from "./src/hsv/types";
-export type { Hwb } from "./src/hwb/types";
-export type { Jab } from "./src/jab/types";
-export type { Jch } from "./src/jch/types";
-export type { Lab } from "./src/lab/types";
-export type { Lab65 } from "./src/lab65/types";
-export type { Lch } from "./src/lch/types";
-export type { Lch65 } from "./src/lch65/types";
-export type { Lchuv } from "./src/lchuv/types";
-export type { Lrgb } from "./src/lrgb/types";
-export type { Luv } from "./src/luv/types";
-export type { Okhsl } from "./src/okhsl/types";
-export type { Okhsv } from "./src/okhsv/types";
-export type { Oklab } from "./src/oklab/types";
-export type { Oklch } from "./src/oklch/types";
-export type { P3 } from "./src/p3/types";
-export type { Prophoto } from "./src/prophoto/types";
-export type { Rec2020 } from "./src/rec2020/types";
-export type { Rgb } from "./src/rgb/types";
-export type { Xyb } from "./src/xyb/types";
-export type { Xyz50 } from "./src/xyz50/types";
-export type { Xyz65 } from "./src/xyz65/types";
-export type { Yiq } from "./src/yiq/types";
+export type * from "./src/index.js";
```

Comment:
Types were moved to `src/index.d.ts` to also mirror the culori package

Reply:
This change might cause the same issue as discussed [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72323#discussion_r2051524011).

Reply:
Also responded in other convo, but fixed!